libraries/Ethernet/examples/WebClientRepeatingManual/WebClientRepeatingManual.ino

```diff
@@ -0,0 +1,142 @@
+/*
+  Repeating Web client
+
+ This sketch connects to a a web server and makes a request
+ using a Wiznet Ethernet shield. You can use the Arduino Ethernet shield, or
+ the Adafruit Ethernet shield, either one will work, as long as it's got
+ a Wiznet Ethernet module on board.
+
+ This example uses DNS, by assigning the Ethernet client with a MAC address,
+ IP address, and DNS address.
+ */
+
+#include <ZephyrClient.h>
+#include "ZephyrEthernet.h"
+
+// The IP address will be dependent on your local network.
+// DNS, gateway and subnet are optional:
+IPAddress ip(192, 168, 1, 177);
+IPAddress myDns(192, 168, 1, 1);
+IPAddress gateway(192, 168, 1, 1);
+IPAddress subnet(255, 255, 255, 0);
+
+// initialize the library instance:
+ZephyrClient client;
+
+char server[] = "www.arduino.cc"; // also change the Host line in httpRequest()
```

Comment:
It is no more possible to reach www.arduino.cc using plain http. For this purpose is it better to switch to arduino.tips and get the asciilogo.txt file